doc/developers/develop.rst

```diff
@@ -246,7 +246,8 @@ whether it is just for you or for contributing it to scikit-learn, there are
 several internals of scikit-learn that you should be aware of in addition to
 the scikit-learn API outlined above. You can check whether your estimator
 adheres to the scikit-learn interface and standards by running
-:func:`utils.estimator_checks.check_estimator` on the class::
+:func:`utils.estimator_checks.check_estimator` on the class (see its docstring
+for details and possible interactions with `pytest`)::
```

Comment:
```suggestion
:func:`utils.estimator_checks.check_estimator` on the class or using :func:`~sklearn.utils.parametrize_with_checks` pytest decorator 
(see its docstring for details and possible interactions with `pytest`)::
```
OK with removing the detailed example, but I think it's important to mention `parametrize_with_checks` because it's more useful for contrib projects and it's harder to find otherwise.